### PR TITLE
add role & aria-labels for accessibility

### PR DESCRIPTION
**Pull request by vanpertsch**
Signed-off-by: Vanessa Pertsch <vanessa.pertsch@posteo.de>

### CODE REVIEW

**Comment by PVince81** [comment]
@artonge @Pytal please review / take over to get this merged (needs rebase)

**Comment by nickvergessen** [comment]
Ready to review

**Review by Pytal — APPROVED** [review]

File: src/components/Modal/Modal.vue [hunk 1]
```diff
@@ -195,7 +200,7 @@
 					</transition>
 
 					<!-- Content -->
-					<div class="modal-container">
+					<div id="modal-description" class="modal-container">
```
Comment (vinicius73) [inline src/components/Modal/Modal.vue:203]: Can we use a dynamic id here?

**Review by vinicius73 — APPROVED** [review]

**Comment by nickvergessen** [comment]
There is still no stable5 branch, but i guess we should create it from 5.3.x

**Comment by juliushaertl** [comment]
There is now, see #2907

**Comment by nickvergessen** [comment]
I'll backport it manually

**Comment by raimund-schluessler** [comment]
The ids might not be unique if there are multiple modals on the page. I think this needs to be enhanced with a randomly generated string,

**Comment by nickvergessen** [comment]
The modals are not mounted until displayed. So should be fairly save for now?

**Comment by raimund-schluessler** [comment]
> The modals are not mounted until displayed. So should be fairly save for now?

Depends. I think I saw a case in an app where two modals appeared on top of each other. Don't remember where, though.

**Pull request merged**